app/views/shared/_article_fields_price.html.haml

```diff
@@ -1,3 +1,7 @@
+.fold-line
+  .split-price-warning
+    .alert.alert-warning Supplier price does not equal (Price * Unit).  If this is not intended use the 'Recalculate Split Price' button to update the price.
```

Comment:
Double space in this line.
I guess you mean _price * unit_quantity_?
Let's also use i18n here.

Reply:
i've replaced with i18n.

as to the term, yes, technically correct,  however the UI doesn't show _unit_quantity_  rather it shows "Unit"